x/wasm/keeper/querier_test.go

```diff
@@ -871,17 +871,39 @@ func TestQueryParams(t *testing.T) {
 	require.Equal(t, paramsResponse.Params.CodeUploadAccess, defaultParams.CodeUploadAccess)
 	require.Equal(t, paramsResponse.Params.InstantiateDefaultPermission, defaultParams.InstantiateDefaultPermission)
```

Comment:
I think you can delete from L865-L872, because it is the test about DefaultParams.

Reply:
I fixed it.